BuildForSDGCohort2/Team-030-Backend
# Project folders

### About PR
This PR sets up script automation: lint code using the Airbnb style guide.

**Testing steps:** 
```js
npm install
npm start 
```

The script starts and the linting rules are executed to check through the code.
For fast linting error fix, 
``` 
npm run fix
```
This will run a quick fix of some codes and show the status of the code: either CLEAN or Error

**Relevant Files**
.eslintrc
package.json

Issue
#17
